engine/src/main/java/org/hibernate/validator/internal/properties/ConstrainableType.java

```diff
@@ -0,0 +1,14 @@
+/*
+ * Hibernate Validator, declare and validate application constraints
+ *
+ * License: Apache License, Version 2.0
+ * See the license.txt file in the root directory or <http://www.apache.org/licenses/LICENSE-2.0>.
+ */
+package org.hibernate.validator.internal.properties;
+
+/**
+ * @author Marko Bekhta
+ */
+public interface ConstrainableType {
```

Comment:
What is this doing?

Reply:
For now, nothing. I think Marko wanted a common interface for the `JavaBean` and the upcoming `PropertyHolder`.

Let's add a comment.